source/math/FloatExtension.ooc

```diff
@@ -126,17 +126,17 @@ extend Float {
 		}
 		result
 	}
-	getRadix: static func (value: Float, valueDigits: Int) -> Float {
+	getRadix: static func (value: This, valueDigits: Int) -> This {
 		(tempValue, result) := This decomposeToCoefficientAndRadix(value, valueDigits)
 		result
 	}
-	getScientificPowerString: static func (value: Float) -> String {
+	getScientificPowerString: static func (value: This) -> String {
 		(coefficient, radix) := This decomposeToCoefficientAndRadix(value, 1)
 		power := log10(radix) as Int
 		result := ""
-		if (coefficient != 1.0f)
-			result = result & coefficient toString() & "×" clone()
-		result = result & "10^" clone() & power toString()
+		if (This absolute(coefficient - 1.0f) > This epsilon)
```

Comment:
I still think this is unnecessary. You should only need to check that `This absolute(coefficient) != 1.0f`. If the difference between `coefficient` and `1.0f` is smaller than `This epsilon`, then `coefficient` **is** equal to 1.0f.